ksp_plugin/interface_body.hpp

```diff
@@ -197,6 +208,49 @@ FromAdaptiveStepParameters(
       adaptive_step_parameters.speed_integration_tolerance * (Metre / Second));
 }
 
+inline std::pair<
+    physics::Ephemeris<Barycentric>::AdaptiveStepParameters,
+    physics::Ephemeris<Barycentric>::GeneralizedAdaptiveStepParameters>
+FromFlightPlanAdaptiveStepParameters(FlightPlanAdaptiveStepParameters const&
+                                         flight_plan_adaptive_step_parameters) {
+  serialization::AdaptiveStepSizeIntegrator message1;
+  CHECK(serialization::AdaptiveStepSizeIntegrator::Kind_IsValid(
+      flight_plan_adaptive_step_parameters.integrator_kind));
+  message1.set_kind(
+      static_cast<serialization::AdaptiveStepSizeIntegrator::Kind>(
+          flight_plan_adaptive_step_parameters.integrator_kind));
```

Comment:
Move the definitions of max_steps, length_integration_tolerance, and speed_integration_tolerance to the beginning, and scope message1 and message2 to avoid mixing them up (note: if they are scoped, they can both be called message).

Reply:
As discussed, no.